src/routes/Board/Board.tsx

```diff
@@ -9,7 +9,9 @@ import {useAppSelector} from "store";
 function Board() {
   const state = useAppSelector((applicationState) => ({
     board: applicationState.board,
-    notes: applicationState.notes,
+    notes: applicationState.notes.filter(
+      (note) => applicationState.board.data?.showNotesOfOtherUsers || (!applicationState.board.data?.showNotesOfOtherUsers && Parse.User.current()?.id === note.author)
```

Comment:
```suggestion
      (note) => applicationState.board.data?.showNotesOfOtherUsers || Parse.User.current()?.id === note.author
```

Shouldn't this be enough? Show the note if it's either your own or all notes are shown

Reply:
Correct. This should be enough.